tests/lib/c_lib/src/main.c

```diff
@@ -335,7 +653,28 @@ void test_main(void)
 			 ztest_unit_test(test_strlen),
 			 ztest_unit_test(test_strcmp),
 			 ztest_unit_test(test_strxspn),
-			 ztest_unit_test(test_bsearch)
+			 ztest_unit_test(test_bsearch),
+			 ztest_unit_test(test_abs),
+			 ztest_unit_test(test_atoi),
+			 ztest_unit_test(test_isalnum),
+			 ztest_unit_test(test_isalpha),
+			 ztest_unit_test(test_isdigit),
+			 ztest_unit_test(test_isgraph),
+			 ztest_unit_test(test_isprint),
+			 ztest_unit_test(test_isspace),
+			 ztest_unit_test(test_isupper),
```

Comment:
Things might be a little simpler and the test might run a bit faster (especially on our newer cycle-exact simulators where CPU time is a lot more expensive in real world time) if you squished these down to just a few tests, like "test_libc_chartype()", "test_libc_memstr()", etc...

Reply:
OK, I will integrating those tests to a few tests! Thanks